package.json

```diff
@@ -1,10 +1,10 @@
 {
   "private": true,
   "scripts": {
-    "version-stamp": "lerna run version-stamp",
+    "bootstrap": "lerna bootstrap --force-local",
     "build": "lerna run build --no-private",
-    "build:all": "lerna run build",
-    "bootstrap": "lerna bootstrap",
+    "rebuild": "npm run clean && npm clean-install && npm run build",
```

Comment:
Should it be "npm run clean && npm clean-install && npm run bootstrap && npm run build"?
If we are developing and making changes in one of the taquito packages, bootstrap allows us to take the local changes.

Reply:
I didn't see the `"postinstall": "npm run bootstrap"`

Reply:
I just saw this.
Yes, postinstall runs intelligently, and not just post install 😁 (also after e.g. clean-install).